drivers/include/periph/hwrng.h

```diff
@@ -20,6 +20,16 @@
  * @note    Refer to your platforms MCU reference manual for information on the
  *          quality of the used (pseudo) random number generator!
  *
+ * @section     sec_pm (Low-) power implications
+ *
+ * The HWRNG implementation **should** consume no additional power while no read
+ * operation is in progress. This means, that the HWRNG peripheral should be
+ * disabled (e.g. through peripheral clock gating) after the initialization and
+ * that it **should** only be turned on while hwrng_read() is active.
+ *
```

Comment:
well, maybe the **should** is important here. HWRNG's might have an internal buffer which slowly fills in the background.